tests/unit/stripe/test_assembler.py

```diff
@@ -108,15 +103,18 @@ def it_knows_the_rows_dimension_fills(
             instance_mock(request, _Element, fill=fill)
             for fill in ("cdef01", "6789ab", "012345")
         )
+        rows_dimension_.subtotals = tuple(
+            instance_mock(request, _Subtotal, fill=fill) for fill in ("STF_1", "STF_2")
+        )
         _row_order_prop_.return_value = np.array([2, -2, 1, -1, 0])
         assembler = StripeAssembler(None, rows_dimension_, None, None)
 
         print(assembler.rows_dimension_fills)
         assert assembler.rows_dimension_fills == (
             "012345",
-            None,
+            "STF_2",
             "6789ab",
-            None,
+            "STF_1",
```

Comment:
I'm not sure this is correct. if the subtotals fills are ["STF_1", "STF_2"] and the idx is -2 why the second element of the expectation is `_2` and not `_1`, maybe I'm wrong but this seems weird to me.

Reply:
yeah, I think it's actually correct... because the indexes are assigned starting from the negative value that's closest to zero, and then decreasing... so the payload order would be -1, -2... and then here, since they're flipped explicitly, with reordering transform, they come the other way round...

Reply:
maybe I'm doing confusion ... but does `-1` mean the last item? like the python indexing starting from the bottom?